docs/courseadministration/addcontent/quiz_import.md

```diff
@@ -0,0 +1,63 @@
+![HPI Logo](../../img/HPI_Logo.png)
+
+# Quiz Import from Google Spreadsheets
+
+You can import the quiz structure from Google Spreadsheets.
+All you have to do is the following:
+
+### 1. Make a copy of this template
+We can provide you with a quiz template that you can copy
+:::image type="content" source="../../img/courseadministration/quizmanagement/quiz_template.png" alt-text="How our quiz template looks like":::
+Find the non-to-be-edited quiz structure template [here](https://docs.google.com/spreadsheets/d/1_b8wO9_HlrrHQxagYan1iM6BBK0XtnD6vESxmhnML5s/edit#gid=0)
+
+Remember: *Please, do not edit directly the template, but make a copy of it* - see below how to do it.
+
+:::image type="content" source="../../img/courseadministration/quizmanagement/copy_quiz_template.png" alt-text="How to make a copy of a Google spreadsheet template":::
+
+Now that you have created a copy, you can edit yours with the quiz questions for all the weeks and homeworks in your course.
+Please note that every row after the _Quizname_ rows is a question.
+
+#### Cheat sheet of quiz questions and answers
+
+##### The infos about the quiz and its type:
+
+- *Name* : the name of the quiz
+- *CourseSection*: the section of the course where the quiz is
```

Comment:
We should add a note here that this value is always numeric and that course sections are counted starting with `1`.